packages/cozy-client/src/helpers.spec.js

```diff
@@ -228,6 +229,53 @@ describe('deconstructWebLink', () => {
   )
 })
 
+describe('deconstructRedirectLink', () => {
+  it.each([
+    [
+      'contacts/#/',
+      {
+        slug: 'contacts',
+        path: '',
+        hash: '/'
+      }
+    ],
+    [
+      'contacts/#/hash',
+      {
+        slug: 'contacts',
+        path: '',
+        hash: '/hash'
+      }
+    ],
+    [
+      'contacts/#/long/hash',
+      {
+        slug: 'contacts',
+        path: '',
+        hash: '/long/hash'
+      }
+    ],
+    [
+      'contacts/path/#/long/hash',
+      {
+        slug: 'contacts',
+        path: 'path/',
+        hash: '/long/hash'
+      }
+    ],
+    [
+      'contacts/long/path/#/long/hash',
+      {
+        slug: 'contacts',
+        path: 'long/path/',
+        hash: '/long/hash'
+      }
```

Comment:
@Crash-- What do you think about the new version ? Is it okay to have a trailing slash at the end of path ?